contexts/credentials/v2

```diff
@@ -4,7 +4,14 @@
     "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#",
 
     "id": "@id",
+    "kid": "@id",
+    "iss": "@id",
+    "sub": "@id",
+    "jku": "@id",
+    "x5u": "@id",
```

Comment:
```suggestion
    "kid": "@id",
    "iss": "@id",
    "sub": "@id",
    "jku": "@id",
    "x5u": "@id",
```

The code above aliases everything to the `@id` JSON-LD keyword, which will lead to every JWT property listed above being mapped back to the `id` property... which is probably not what you wanted.

Reply:
https://github.com/w3c/vc-data-model/pull/1149/files#diff-267b7e47789daf9a1d312b42826aa937a1bb5af4312ee90df6b8a64cd6426f4fR21

Ahh I meant to do this:

```
 "issuer": {
          "@id": "https://www.w3.org/2018/credentials#issuer",
          "@type": "@id"
        },
```

Reply:
@OR13 wrote:
> Ahh I meant to do this

Ok, thanks for fixing that... the PR makes a lot more sense now.